python-sdk/nuscenes/predict/tests/run_mtp.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+import torch
+import numpy as np
+from torch.utils.data import DataLoader, IterableDataset
+import torch.optim as optim
+
+from nuscenes.predict.models.mtp import ResNetBackbone, MTP, MTPLoss
+
+class TestDataset(IterableDataset):
```

Comment:
Let's add a comment for what this test does

Reply:
I've changed `TestDataset` to `Dataset` and added a module docstring explaining what this script does.